src/lib/xmlparse.rs

```diff
@@ -1432,15 +1375,16 @@ pub unsafe extern "C" fn XML_ParserCreate_MM(
     mut memsuite: Option<&XML_Memory_Handling_Suite>,
     mut nameSep: *const XML_Char,
 ) -> XML_Parser {
-    XML_ParserStruct::create(encodingName, memsuite, nameSep, NULL as *mut DTD)
+    // FIXME
```

Comment:
I'm thinking about returning a null pointer here instead, or panic.

Reply:
I was tempted to use [expect_none](https://doc.rust-lang.org/std/option/enum.Option.html#method.expect_none) but it's nightly-only and guarded by a feature.

Reply:
I switched to `unimplemented!`.